xorg_xmessage.rb

```diff
@@ -0,0 +1,18 @@
+require 'package'
+
+class Xorg_xmessage < Package
+  description 'Xmessage displays a message or query in a window.'
+  homepage 'https://gitlab.freedesktop.org/xorg/app/xmessage'
+  version 'e315069848aea3aaeb72bd3bc9a5d0bdb1242ed7'
+  source_url 'https://gitlab.freedesktop.org/xorg/app/xmessage/-/archive/e315069848aea3aaeb72bd3bc9a5d0bdb1242ed7/xmessage-e315069848aea3aaeb72bd3bc9a5d0bdb1242ed7.tar.gz'
```

Comment:
Use this instead: https://www.x.org/releases/individual/app/xmessage-1.0.5.tar.bz2